roadmap.md

```diff
@@ -1,87 +1,27 @@
 # Confidential Containers Roadmap
-When looking at the project's roadmap we distinguish between the short-term roadmap (2-4 months) vs. 
-the mid/long-term roadmap (4-12 months):
-- The **short-term roadmap** is focused on achieving an end-to-end, easy to deploy confidential 
-  containers solution using at least one HW encryption solution and integrated to k8s (with forked
-  versions if needed)
-- The **mid/long-term solutions** focuses on maturing the components of the short-term solution 
-  and adding a number of enhancements both to the solution and the project (such as CI,
-  interoperability with other projects etc.)
 
-# Short-Term Roadmap
-The short-term roadmap aims to achieve the following:
-- MVP stack for running confidential containers
-- Based on and compatible with Kata Containers 2
-- Based on at least one confidential computing implementation (SEV, TDX, SE, etc)
-- Integration with Kubernetes: kubectl apply -f confidential-pod.yaml
-
-The work is targeted to be completed by end of November 2021 and includes 3 milestones:
-![September 2021](./images/RoadmapSept2021.jpg)
```

Comment:
the images could be `git rm`d

Reply:
Good point

Reply:
:-) I just updated PR with their removal ...